provider/datagen/src/transform/cldr/displaynames/region.rs

```diff
@@ -0,0 +1,134 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
I also suspect you added region.rs without deleting territory.rs

Reply:
Yes. Forgot to commit the change. Removed now.